commit 7af3dfd410d5c69b34fdc839acd5e395ba0c0369
Author: howetuft <howetuft@gmail.com>
Date:   Tue Aug 13 13:53:15 2024 +0200

    Fix FMT_INCLUDES

diff --git a/src/cmake/externalpackages.cmake b/src/cmake/externalpackages.cmake
index 2c296b712..deef078ff 100644
--- a/src/cmake/externalpackages.cmake
+++ b/src/cmake/externalpackages.cmake
@@ -318,4 +318,5 @@ macro (find_or_download_fmt)
 endmacro()
 
 find_or_download_fmt()
+set(FMT_INCLUDES ${fmt_INCLUDE_DIR})
 include_directories (${FMT_INCLUDES})
